Pod/Classes/SocketManager.swift

```diff
@@ -92,16 +98,21 @@ public  class SocketManager {
                         self.experienceChannel?.onRequest(response)
                         self.organizationChannel?.onRequest(response)
                     }else{
-                        let channelEnum:SOCKET_CHANNELS = SOCKET_CHANNELS(rawValue: channel as! String)!
-                        switch channelEnum{
-                            case .SYSTEM :
-                                self.systemChannel?.onRequest(response)
-                            case .LOCATION:
-                                self.locationChannel?.onRequest(response)
-                            case .EXPERIENCE:
-                                self.experienceChannel?.onRequest(response)
-                            case .ORGANIZATION:
-                                self.organizationChannel?.onRequest(response)
+                        if(self.channelCache.indexForKey(channel as! String) != nil){
```

Comment:
I think the order of operations here lets developers overwrite the system (and other built in channels), it should probably check the  SOCKET_CHANNELS before it checks the channelCache right?